stat/stat_unix1.go

```diff
@@ -0,0 +1,19 @@
+// +build linux solaris
+
+package stat
```

Comment:
Just leave the existing duplication alone.

Sorry for being particular here, but this kind of package can have the tendency to become problematic over time.